escrow/scripts/leaderboards/read_leaderboard_info.cdc

```diff
@@ -0,0 +1,13 @@
+import Escrow from "../../contracts/AllDay.cdc"
+
+// This script returns the leaderboard info for the given leaderboard name.
+pub fun main(leaderboardName: String, address: Address): Escrow.LeaderboardInfo? {
```

Comment:
the address doesn't need to be an argument. Should be the same address `Escrow` is imported from

Reply:
Ah I see - I was confused because of the import and not having it be `0x`. I forgot we had to translate it for the script. Ty!